Dockerfile

```diff
@@ -6,62 +6,57 @@ RUN apt-get update && apt-get install -y \
     libhdf5-dev \
     libgeos-dev
 
-RUN pip install numpy \
-    cython
-RUN pip install -v scipy
+RUN pip install numpy cython scipy
 RUN pip install scikit-learn pandas h5py matplotlib
-RUN pip install https://github.com/gallantlab/pycortex/archive/fe58400c8c3a3187d930b8a696cda8fec62c0f19.zip --egg
-RUN pip install uwsgi
-RUN pip install 'Django==1.8.8'
-RUN pip install 'python-social-auth==0.2.13'
+
+RUN pip install celery[redis]
+RUN pip install certifi==2015.04.28
+RUN pip install cognitiveatlas
+RUN pip install Django==1.8.8
 RUN pip install djangorestframework
-RUN pip install markdown
+RUN pip install django-celery
+RUN pip install django-chosen
+RUN pip install django-cleanup==0.4.2
+RUN pip install django-coffeescript
+RUN pip install django-crispy-forms
+RUN pip install django-datatables-view
+RUN pip install django-dbbackup<2.3
```

Comment:
Aw, so sorry for that... I tried a pip install without quotations and it worked, so I though it would work for all of them.